File: iosApp/iosApp/feature/profile/payment/PaymentMethodUi.swift
//
//  PaymentMethodUi.swift
//  iosApp
//
//  Created by Марк Шавловский on 22.06.2024.
//  Copyright © 2024 orgName. All rights reserved.
//

import Foundation
import SwiftUI

struct PaymentMethodUI {
    var uuid: String
    var name: LocalizedStringKey
    var value: PaymentMethodValueUI?
}

struct PaymentMethodValueUI {
    var value: String
    var valueToCopy: String
}

File: shared/src/commonMain/kotlin/com/bunbeauty/shared/ui/screen/profile/screen/payment/PaymentMethodUI.kt
package com.bunbeauty.shared.ui.screen.profile.screen.payment

data class PaymentMethodUI(
    val uuid: String,
    val name: String,
    val value: PaymentMethodValueUI?,
)

data class PaymentMethodValueUI(
    val value: String,
    val valueToCopy: String,
)
